src/Plugin/OpenIDConnectClient/OpenIDConnectB2CClient.php

```diff
@@ -75,16 +74,60 @@ public function submitConfigurationForm(array &$form, FormStateInterface $form_s
   public function getClientScopes(): ?array {
     return $this->configuration['scopes'];
   }
+
   /**
    * {@inheritdoc}
    */
   public function getEndpoints() : array {
     return [
-      'authorization' => 'https://' . $this->configuration['tenant'] . '.b2clogin.com/' . $this->configuration['tenant']. '.onmicrosoft.com/oauth2/v2.0/authorize?p=' . $this->configuration['flow'],
-      'token' => 'https://' . $this->configuration['tenant'] . '.b2clogin.com/' . $this->configuration['tenant']. '.onmicrosoft.com/oauth2/v2.0/token?p=' . $this->configuration['flow'],
+      'authorization' => 'https://' . $this->configuration['tenant'] . '.b2clogin.com/' . $this->configuration['tenant'] . '.onmicrosoft.com/oauth2/v2.0/authorize?p=' . $this->configuration['flow'],
+      'token' => 'https://' . $this->configuration['tenant'] . '.b2clogin.com/' . $this->configuration['tenant'] . '.onmicrosoft.com/oauth2/v2.0/token?p=' . $this->configuration['flow'],
       'userinfo' => '',
-      'end_session' => 'https://' . $this->configuration['tenant'] . '.b2clogin.com/' . $this->configuration['tenant']. '.onmicrosoft.com/oauth2/v2.0/logout?p=' . $this->configuration['flow'],
+      'end_session' => 'https://' . $this->configuration['tenant'] . '.b2clogin.com/' . $this->configuration['tenant'] . '.onmicrosoft.com/oauth2/v2.0/logout?p=' . $this->configuration['flow'],
     ];
   }
 
-}
\ No newline at end of file
+  /**
+   * Parse the token from upstream.
+   */
```

Comment:
Not sure why CodeSniffer didn't spot this missing for doc
```
@param string $token
  Token.
```

Reply:
Fixed this and the other instance.